routing/routes.go

```diff
@@ -1049,6 +1049,27 @@ func NewRoutes(conf *config.Config, kialiCache cache.KialiCache, clientFactory k
 			handlers.NamespaceTls,
 			true,
 		},
+		// swagger:route GET /cluster/tls tls ClusterTls
+		// ---
+		// Get TLS statuses for given namespaces of the given cluster
+		//
+		//     Produces:
+		//     - application/json
+		//
+		//     Schemes: http, https
+		//
+		// responses:
+		//      200: clusterTlsResponse
+		//      400: badRequestError
+		//      500: internalError
+		//
+		{
+			"ClusterTls",
+			"GET",
+			"/api/cluster/tls",
+			handlers.ClusterTls,
+			true,
+		},
```

Comment:
I will leave them separately not to complicate the logic, and the `/namespaces/tls` is not used in UI anymore, but we can leave it as REST API call for the community users.

Reply:
I would remove it so as not to cause confusion later but that can be done separately. The API is considered internal only to kiali. If there's e2e tests that cover the `/namespaces/tls` endpoint we should get those moved over to `/cluster/tls`. I think we can do some cleanup in general of the API routes. There's already an `/api/clusters` endpoint and now were adding `/api/cluster`. We could even add more to `/api/cluster` as we continue to optimize the frontend calls. Because the api is considered internal only, it's not as much of a concern but it'd be nice to be consistent and clean things up at some point.

Reply:
good point to have single `/clusters**`